src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java

```diff
@@ -870,6 +870,32 @@ public void publishActivity(boolean refreshStream) {
     }
   }
 
+  public void attachImagesToActivity() {
+    waitCKEditorLoading(OPENED_ACTIVITY_COMPOSER_DRAWER_SELECTOR);
+    ckEditorAttachImageButton().assertVisible();
+    ElementFacade fileInput = ckEditorAttachImageInput();
+    fileInput.assertEnabled();
+    attachImageToFileInput(fileInput);
+    activityDrawerAttachedImagesCarousel().assertVisible();
```

Comment:
We may need to ensure that `loading` effect isn't displayed before making this assertion.